inc/mlx90632.h

```diff
@@ -239,6 +239,31 @@ double mlx90632_calc_temp_object(int32_t object, int32_t ambient,
                                  int32_t Ea, int32_t Eb, int32_t Ga, int32_t Fa, int32_t Fb,
                                  int16_t Ha, int16_t Hb);
 
+/** Calculation of object temperature when the environment temperature differs from the sensor temperature
+ *
+ * DSPv5 implementation of object temperature calculation with customer
+ * calibration data
+ *
+ * @param[in] object object temperature from @link mlx90632_preprocess_temp_object @endlink
+ * @param[in] ambient sensor ambient temperature from @link mlx90632_preprocess_temp_ambient @endlink
+ * @param[in] reflected reflected (environment) temperature from a sensor different than the MLX90632 or acquired by other means
+ * @param[in] Ea Register value on @link MLX90632_EE_Ea @endlink
+ * @param[in] Eb Register value on @link MLX90632_EE_Eb @endlink
+ * @param[in] Ga Register value on @link MLX90632_EE_Ga @endlink
+ * @param[in] Fb Register value on @link MLX90632_EE_Fb @endlink
+ * @param[in] Fa Register value on @link MLX90632_EE_Fa @endlink
+ * @param[in] Ha Register value on @link MLX90632_EE_Ha @endlink
+ * @param[in] Hb Register value on @link MLX90632_EE_Hb @endlink
+ *
+ * @note emissivity Value provided by user of the object emissivity
+ * using @link mlx90632_set_emissivity @endlink function.
+ *
+ * @return Calculated object temperature in milliCelsius
+ */
+double mlx90632_calc_temp_object_reflected(int32_t object, int32_t ambient, double reflected,
```

Comment:
are we sure reflected is proper term for general usage? Maybe `mlx90632_calc_temp_object_enviornment` , but I also don't like that much. Any other suggestions you had?